File: src/main/java/com/p2p/lending/service/NoticeService.java
package com.p2p.lending.service;

import java.util.List;

import com.p2p.lending.pojo.Notice;

public interface NoticeService {

	public List<Notice> noticelist(String ids);

	public Notice noticeget(Integer ids);

	public void noticeadd(Notice notice);

	public void noticeupd(Notice notice);

	public void noticedel(Integer ids);
	
	public void noticeupds(Notice notice);
	
	public void noticshiji(Integer ids);

	public List<Notice> noticetop5();
	public List<Notice> noticetop5meiti();
	public List<Notice> noticetop5sy();

	
}

File: src/main/java/com/p2p/lending/service/NoticeService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Notice
import org.springframework.stereotype.Service

@Service
interface NoticeService {

    fun noticelist(ids: String): List<Notice>

    fun noticeget(ids: Int?): Notice

    fun noticeadd(notice: Notice)

    fun noticeupd(notice: Notice)

    fun noticedel(ids: Int?)

    fun noticeupds(notice: Notice)

    fun noticshiji(ids: Int?)

    fun noticetop5(): List<Notice>
    fun noticetop5meiti(): List<Notice>
    fun noticetop5sy(): List<Notice>


}
